frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/StatusField.tsx

```diff
@@ -122,6 +130,17 @@ export function StatusField(props: { cluster: Cluster }) {
             )
             break
         case ClusterStatus.creating:
+            if (isHybrid) {
```

Comment:
I think this needs to be done in each case when a button with 'launchLogs' is shown. So on line 147. Also note that the 'View Logs' button is also shown in 'HiveNotification' component which is displayed when a cluster installation fails:

<img width="1287" alt="Screenshot 2022-03-10 at 11 40 17" src="https://user-images.githubusercontent.com/1121740/157645034-c2017be7-37dc-4a8b-868f-78546054f6e5.png">

Reply:
I will look for all the places where the 'View Logs' button is and I will replace it with the "Download logs" button (only for assisted clusters)

Reply:
@jtomasek It makes sense to change the launchLogs method of https://github.com/stolostron/console/blob/main/frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx class to display the link to view logs or download logs depending on the type of cluster? or is it better to add a conditional to each class where the launchlogs method is found?